### PR TITLE
Generate QRCode using Python library

### PR DESCRIPTION
**Pull request by andykim**
## Description
- Generate QRCode using Python library
https://xendit.atlassian.net/browse/TPI-8122

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] General improvement (non-breaking feature that might not necessarily impact customer)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code (if necessary), particularly in hard-to-understand areas
- [ ] My changes generate no new **vulnerabilities**
- [ ] I have added tests that prove my fix is effective or that my feature works

## Merging
- [ ] I am aware which environment I intend to merge (staging | production)
- [ ] I am aware that a deployment will be triggered if I merge to staging and release branch
- [ ] I informed the team about my merge if it has risk of breaking the target API

## Deployment
- [ ] Unit tests passed
- [ ] UAT passed and communicated to QA and Product is informed about this deployment
- [ ] I'm aware off all the commits / merges behind that will be included in this deployment
- [ ] Feature flags are ready and communicated with the team

Please follow our [deployment guideline](https://xendit.atlassian.net/wiki/spaces/TPI/pages/2331476248/Engineer+Onboarding#Releasing-Flow).

## Screenshot

[Screencast from 26-09-2022 15:35:59.webm](https://user-images.githubusercontent.com/5335952/192231292-39878a76-0ab5-476d-bc2b-efaecba26c2b.webm)

### CODE REVIEW

File: xendit_pos/models/xendit_client.py [hunk 1]
```diff
@@ -71,6 +72,8 @@ def create_invoice(self, payment_method, data):
                 res.status_code
             )
 
+        # generate qrcode
+        response['qrcode_image'] = self.qrCode.renderQrcode(response['invoice_url'])
```
Comment (yanoandri) [inline xendit_pos/models/xendit_client.py:76]: interesting, what is the difference between the old version from this one? @andykim

Comment (andykim) [reply xendit_pos/models/xendit_client.py:76]: @yanoandri In the current version we generate the QRcode from a third-party service https://github.com/xendit/odoo-pos-module/blob/d8b5a7aa8bbc9bccd85290b2542d3c134c6ae66d/xendit_pos/static/src/js/payment_xendit_pos.js#L199

File: xendit_pos/utils/qrcode.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            error_correction = qrcode.constants.ERROR_CORRECT_L,
+            box_size = 10,
+            border = 4,
+        )  
+
+    def generateQrCode(self, xendit_invoice_url):
+        qrcode = self.initQrCode()
+        qrcode.add_data(xendit_invoice_url)
+        qrcode.make(fit = True)
+        qrcode_image = qrcode.make_image(fill_color = "black", back_color = "transparent")
```
Comment (ikanyu) [inline xendit_pos/utils/qrcode.py:22]: 😬 let's keep the changes of single and double quote consistent in the changes.

**Review by ikanyu — APPROVED** [review]
One small comment else LGTM

File: xendit_pos/utils/qrcode.py (hunk 2 above)
Comment (andykim) [reply xendit_pos/utils/qrcode.py]: :D forgot it

**Pull request merged**